.travis.yml

```diff
@@ -77,19 +77,19 @@ jobs:
       script:
         - unbuffer node build-system/pr-check/remote-tests.js
       after_script:
-        - build-system/sauce_connect/stop_sauce_connect.sh
```

Comment:
This was intentionally duplicated because when a test task fails, the sauce connect proxy [isn't stopped](https://travis-ci.org/ampproject/amphtml/jobs/547874724#L2081-L2084) by the Travis job due to `execOrDie`. When tests pass, the second `after_script` call is a no-op.